### DISCUSSION TITLE
Is there any way to have a listing of all papers from a Bibtex file!

### DISCUSSION THREAD

**Discussion by shravanngoswamii** [quarto-dev/quarto-cli#10330] [Q&A]
### Description

I have a file mlg.bib, we were generating HTML pages authorwise, categorywise and yearwise using custom shell script and then generating this site: https://mlg.eng.cam.ac.uk/pub/ using Hugo, but now we want to switch to Quarto. Is there any simple way for doing this!

**Comment by mcanouil** [comment]
This has been discussed already in at least two places easily discoverable by search.

- https://github.com/quarto-dev/quarto-cli/issues/1324
- https://github.com/quarto-dev/quarto-cli/issues/2925

Please note that you have a lot of information already there between issues and discussions which are searchable. You can also leverage labels to filter down results.
